packages/perl_data_dumper.rb

```diff
@@ -3,23 +3,23 @@
 class Perl_data_dumper < Package
```

Comment:
No buildsystems/perl?

Reply:
The more complicated packages I did not change.

Reply:
changed